dataland-frontend/tests/e2e/fixtures/lksg/LksgDataFixtures.ts

```diff
@@ -48,15 +49,15 @@ export function generateProductionSite(undefinedProbability = 0.5): LksgProducti
  * @returns 1 to 5 random production sites
  */
 export function generateArrayOfProductionSites(): LksgProductionSite[] {
-  return Array.from({ length: faker.datatype.number({ min: 0, max: 5 }) }, generateProductionSite);
+  return Array.from({ length: faker.datatype.number({ min: 1, max: 5 }) }, generateProductionSite);
```

Comment:
Why is it not possible to have no production site?

Reply:
It is. But how it was implemented before a fixture could have "Yes" selected for "Production sites" but provide an empty list of production sites which is not allowed. Now every fixture has a production site which is technically nonsensical but the frontend and tests work fine with it.

Reply:
ok, after a call we agreed to include a test that uploads a dataset with all Yes/No fields set to "no".
To give 0 as the number of production sites does not make sense in a way, that only the formkit validation will be tested and it does not provide really more value to our testing setup